dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatHistory.cs

```diff
@@ -69,6 +70,27 @@ public ChatHistory(IEnumerable<ChatMessageContent> messages)
     public void AddMessage(AuthorRole authorRole, ChatMessageContentItemCollection contentItems, Encoding? encoding = null, IReadOnlyDictionary<string, object?>? metadata = null) =>
         this.Add(new ChatMessageContent(authorRole, contentItems, null, null, encoding, metadata));
 
+    /// <summary>
+    /// Adds <see cref="ChatMessageContent"/> with content items./>
+    /// </summary>
+    /// <param name="authorRole">Role of the message author</param>
+    /// <param name="items">The content items.</param>
+    [Experimental("SKEXP0001")]
+    public void AddMessage(AuthorRole authorRole, params KernelContent[] items)
```

Comment:
Arent we going with `Add` and not use property or arguments anymore ? 

This might not be needed at all.

Reply:
removed